packages/SwingSet/src/hostStorage.js

```diff
@@ -97,21 +99,15 @@ export function buildHostDBInMemory(storage) {
     // after earlier changes have actually been applied, potentially leaving
     // the store in an indeterminate state.  Problem?  I suspect so...
     for (const c of changes) {
-      if (`${c.op}` !== c.op) {
-        throw new Error(`non-string c.op ${c.op}`);
-      }
-      if (`${c.key}` !== c.key) {
-        throw new Error(`non-string c.key ${c.key}`);
-      }
+      assert(`${c.op}` === c.op, X`non-string c.op ${c.op}`);
```

Comment:
This could probably turn into `assert.typeof`, I was probably unaware of its existence when I wrote this check.

Reply:
Did a bunch